larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -223,144 +222,60 @@ TfLiteStatus Prepare(KernelType kernel_type, const int bitwidth,
          conv_params->dilations[1] /* height */ != 1 ||
          conv_params->filter_width != 1 || conv_params->filter_height != 1);
 
-    // Figure out how many temporary buffers we need
-    int temporaries_count = 0;
```

Comment:
keep the `temporaries_count` variable as before. if we decided to add a new temp buffer we don't need to add it back again.